app/models/item.rb

```diff
@@ -221,6 +221,10 @@ def inventory_item_at(storage_location_id)
     inventory_items.find_by(storage_location_id: storage_location_id)
   end
 
+  def self.existing_item_with_same_name?(organization_id, name)
+    where(organization_id: organization_id, name: name).exists?
```

Comment:
Could we check it in a case-insensitive way?   The issue didn't ask for it, but having two items that are only different in case would be confusing.

Reply:
Good point! I'm on it

Reply:
Updated to check in a case-insensitive way! ✅